science-pipelines/development-tutorial.rst

```diff
@@ -17,11 +17,27 @@ This tutorial shows you a workflow for developing LSST Science Pipelines package
    For a more comprehensive look at the code development workflow for the LSST Science Pipelines, see the `LSST DM Developer Guide <https://developer.lsst.io/work/flow.html>`__.
    This tutorial only covers procedures particular to working in the Notebook Aspect.
 
+.. _eups-prereqs:
+
+Prerequisites
+=============
+
+Log in with a current daily image
+---------------------------------
+
+When you're developing an LSST Science Pipelines package, you're generally working from a branch based on the package's ``master`` branch.
+To ensure that your package can be compiled and used with other LSST Science Pipelines packages, you need to select a recent daily build of the LSST Science Pipelines when you log into the Notebook Aspect.
+
+If you aren't sure you are running the most recent daily, save your work and :doc:`log out </getting-started/logging-out>`.
+Then :doc:`log back in </getting-started/logging-in>` with the most recent daily build.
+
 Set up Git
-==========
+----------
 
 Before you get started with any development, be sure to :doc:`configure Git and set up your GitHub credentials </environment/git-configuration>` (particularly if you will be pushing code).
```

Comment:
Do we need to explicitly say that the first time you use the LSP, you need to say `git lfs install` if you plan on using `git-lfs`?

It's in the dev guide, but we don't link directly to that section in this guide.

Reply:
<img width="657" alt="screen shot 2018-06-05 at 3 30 24 pm" src="https://user-images.githubusercontent.com/349384/41001041-6fad9822-68d5-11e8-8c24-1c6c5e72ce2b.png">

Does this work (from https://nb.lsst.io/environment/git-configuration.html)? nb.lsst.io's Git configuration page basically links you to the Developer Guide for Git LFS.

Reply:
Oh I see. Yes, `git lfs install` is good to note even before worrying about authentication. Will add.

Reply:
I don't have a strong opinion.  I was just pointing out that the section it points to in the dev guide points a few sections down from the [init section](https://developer.lsst.io/git/git-lfs.html#configuring-git-lfs).  If linking there instead makes sense to you, that would be fine, if it doesn't, that's fine with me too.

Reply:
I've got @athornton to get `git lfs install` working automatically, so that solves this.